sense.py

```diff
@@ -8,6 +8,25 @@
 import matplotlib.pyplot as plt
 import cv2
 
+class CSTransform(object):
+
+    def __init__(self, compression_factor, img_shp):
+
+        self.rng = np.random.default_rng(seed=21) #Set RNG for repeatble results
+        self.N =  img_shp[1] *img_shp[2] #length of vectorized image
+        self.M = int(compression_factor * self.N)
+        self.A = 0
+
+    
+    def __call__(self, tensor):
```

Comment:
Proxy image creation with transpose(A)*y should go here. And the Gaussian scheme should probably inherit from CSTransform as well, since everything is the same except A formation

Reply:
Did this for RSTD. Do we want to do it for all the other sensing schemes? I played around with it and it seems to be the exact same if we zero out the rows or use the transpose?

Reply:
Maybe I can do this in a different PR and just put this code in the master branch